src/Components/header/NavHeader/NavHeader.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import '../NavHeader/NavHeader.scss'
+import Button from '../../Button/Button';
+import { useHistory } from "react-router-dom";
```

Comment:
Esto esta raro de leer
```suggestion
import React from 'react';
import { useHistory } from "react-router-dom";
import '../NavHeader/NavHeader.scss'

import Button from '../../Button/Button';

```